src/components/LeafletMap.js

```diff
@@ -250,4 +253,24 @@ export class LeafletMap {
       dispatch.call("hide-loading-indicator");
     }
   };
+  async findAutocompleteLocation(value) {
```

Comment:
@jimboweb you can avoid making the class instance a global on the window if you made this an arrow function.